### PR TITLE
Support directories full of workflows

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
ok, this one is also ready to go
Could we have a new release after this is merged?

File: ephemeris/workflow_install.py [hunk 1]
```diff
@@ -1,33 +1,42 @@
 #!/usr/bin/env python
 import argparse
 import json
+import os
 
 from bioblend import galaxy
 
 
+def import_workflow(gi, path):
```
Comment (afgane) [inline ephemeris/workflow_install.py]: Missing docstring.

File: ephemeris/workflow_install.py [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument("-w", "--workflow_path", help='Path to workflow file')
+    parser.add_argument("-w", "--workflow_path",
+                        help='Path to a workflow file or a directory with multiple workflow files ending with ".ga"')
     parser.add_argument("-g", "--galaxy",
-                        dest="galaxy_url",
-                        help="Target Galaxy instance URL/IP address (required "
-                             "if not defined in the tools list file)",)
-    parser.add_argument("-a", "--apikey",
```
Comment (afgane) [inline ephemeris/workflow_install.py]: Why not leave both options?

**Comment by bgruening** [comment]
@afgane thanks for the review. I added the docstring and the API option back.

**Comment by afgane** [comment]
Thanks for the updates @bgruening

**Comment by bgruening** [comment]
Thanks @afgane!

**Pull request merged**